### PR TITLE
[Doc Improvement][Tab/Bot app display behavior in mobile(iOS/Android)][2678646]

### PR DESCRIPTION
**Pull request by v-spharish**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/design/tabs-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/43098c47eb41ed968856a5239b4c9ba1a6f4a61d/msteams-platform%2Ftabs%2Fdesign%2Ftabs-mobile.md) | 90 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ecd5b878-8331-4447-99d6-55f4edb2ee2b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/design/tabs-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9204b120c67d81a6a336cc8c0665d99b90257259/msteams-platform%2Ftabs%2Fdesign%2Ftabs-mobile.md) | 90 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/677adb0a-42d2-4f05-81ba-dbbba8d7b4bb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/tabs/design/tabs-mobile.md [hunk 1]
```diff
@@ -59,6 +59,9 @@ The following table describes tab availability and behavior when the app is list
 |Channel and group tab|Yes|Tab opens in the device's default browser instead of the Teams mobile client using your app's `websiteUrl` configuration, which must also be included in your source code's `setSettings()` [function](/microsoftteams/platform/tabs/how-to/using-teams-client-sdk#settings-namespace). However, users can view the tab in the Teams mobile client by selecting **More** next to the app and choosing **Open**, which triggers your app’s `contentUrl` configuration.|
 |Personal app|No|Not applicable|
 
+> [!NOTE]
+> If a mobile app has both bot and tab capabilities, bot messages are shown in chat section. When the user selects **More** in the chat of the bot app, tab apps are not shown in that list. However, if user selects **More** from the lower right of the **Chat** section, tab apps are shown with a link to the bot app capability of that app.
```
Comment (surbhigupta12) [inline msteams-platform/tabs/design/tabs-mobile.md:63]: Rewrite. Too wordy, incorrect article usage. Check for grammar.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/design/tabs-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/798cdae936a23fbb85d35fd9376f343096a57707/msteams-platform%2Ftabs%2Fdesign%2Ftabs-mobile.md) | 90 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f40f2cb7-441a-4a31-a017-bff7b9487934) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/design/tabs-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/648898a01a66b33528e013f74106b35bc946e1d7/msteams-platform%2Ftabs%2Fdesign%2Ftabs-mobile.md) | 90 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/02fac134-0833-46b4-b24a-ea117954b6b6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/design/tabs-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a6b06d6ee55973d2df575a355913dd6ffc43027e/msteams-platform%2Ftabs%2Fdesign%2Ftabs-mobile.md) | 90 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a5ba84ce-ab4a-4ed0-b942-e2dad7b7d234) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by SubbaReddi — APPROVED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/design/tabs-mobile.md](https://github.com/MicrosoftDocs/msteams-docs/blob/292a87a85e323e11bca164cbb54ebbacd72122bf/msteams-platform%2Ftabs%2Fdesign%2Ftabs-mobile.md) | 90 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a49445d3-228b-4684-bf6f-555cca692153) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]
